### ISSUE TITLE
support multi-tenancy ?

### ISSUE THREAD

**Issue by freefish1218** [zhenghaoz/gorse#336]
**Is your feature request related to a problem? Please describe.**
every customer has individual data set, we need different recommends result for each customer.

**Describe the solution you'd like**
add multi-tenancy feature

Thanks~

**Comment by zhenghaoz** [comment]
I think multi-tenancy should be implemented by a Kubernetes operator. However, I am still a newcomer in cloud native. Anyway, multi-tenancy could be supported in the future but not now. 😃

**Comment by freefish1218** [comment]
> I think multi-tenancy should be implemented by a Kubernetes operator. However, I am still a newcomer in cloud native. Anyway, multi-tenancy could be supported in the future but not now. 😃

Thank you for your reply!
Maybe I didn't express it clearly, I hope gorse can support the application of SaaS scene.
For example, add a corpid, and each corpid corresponds to a set of data sets, so that each corpid can be recommended separately

**Comment by Doctrs** [comment]
This issue started a year ago. Do you have any plans about multi-tenancy now?

**Comment by zhenghaoz** [comment]
Helm charts are in development: https://github.com/gorse-io/charts

The multi-tenancy will be supported only after the release of Helm charts production version.

**Comment by cevin** [comment]
The SaaS I understand, for example, might be as follows:

```shell
POST /api/user
{
  "Comment": "insect",
  "Labels": [
    "crocodilia",
  ],
  "Subscribe": [
    "fish",
  ],
  "UserId": "cat",
  "TenantId": "tenant_1"  <--
}
```

**Comment by marcospassos** [comment]
Also interested in this feature. Any plans to incorporate it?
